pyproject.toml

```diff
@@ -33,13 +33,26 @@ files = ["*/_version.py"]
 # script = "build.py"  # This gets called as a python script prior to running build steps
 
 [build-system]
+# includes setuptools<70 here in recenter_tc to ensure it is available
+# for akima build
 requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
 build-backend = "poetry_dynamic_versioning.backend"                          # object used to perform the build process
 
 [tool.poetry.dependencies] # must download to run
 python = ">=3.10"       # mandatory to declare the required python version
+# Once meson build is working, use pypi akima86
+# akima86 = "*"
```

Comment:
Note we will want to update this @jsolbrig

Reply:
Yep, agreed. When I work on akima86 and test it, I always do it in coordination with recenter_tc.